features/revision_tags.md

```diff
@@ -0,0 +1,157 @@
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature:
+
+[//]: # (All information in this section is mandatory.)
+
+**Feature name:** Revision tags (stable revision labels)
+
+**Primary lead(s):** Sam Naser
+
+**Short description:** Allows use of revisions with stable namespace labels.
+
+**Design Docs:**  [Initial design](https://docs.google.com/document/d/13IGuJg8swtLdNGW5cpF7ZdVkgge8voNp9DWBD93Wb1Q/edit#heading=h.xw1gqgyqs5b), [Default revision](https://www.google.com/url?sa=t&rct=j&esrc=s&source=appssearch&uact=8&cd=0&cad=rja&q&sig2=5UHj_mlYAbdKKUosJknKZA&ved=0ahUKEwjqq--y-eXxAhUWSWMKHYXzCfY4ABABKAAwAA&url=https://drive.google.com/a/google.com/open?id%3D1P1mNlrJOs-KHVqcKTrFLrvCUiLgyZ-Vlf7ezaZNvD6g%26usp%3Dchrome_omnibox&usg=AOvVaw2PlRZqUGQZejF0NW6RVYB8)
+
+**Relevant Documentation:** [Upgrade documentation](https://istio.io/latest/docs/setup/upgrade/canary/#stable-revision-labels-experimental), [Blog post](https://istio.io/latest/blog/2021/revision-tags/) 
+
+---
+
+## Experimental
+
+### Requirements:
+
+[//]: # (All information in this section is mandatory for promotion. Please modify the links in this)
+[//]: # (section.)
+
+- [x] [User stories]() reviewed in a work group meeting.
+
+- [x] [RFC Authored](https://docs.google.com/document/d/13IGuJg8swtLdNGW5cpF7ZdVkgge8voNp9DWBD93Wb1Q/edit#heading=h.xw1gqgyqs5b)
+
+- [x] [Documentation](https://istio.io/latest/docs/setup/upgrade/canary/#stable-revision-labels-experimental) for enabling and using the feature.
+
+- [ ] [Feedback plan]().
+
+- [x] Disabled by default.
+
+- [x] No impact on performance when the feature is disabled.
+
+---
+
+## Alpha
+
+### Requirements:
+
+**Design**
+
+- [x] RFC has been approved describing the intention of the feature as well as the user stories behind the feature.
+
+**Config**
+
+- [x] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action).
+
+**Docs**
+
+- [x] Reference docs are published to preliminary.istio.io or the Istio wiki.
+- [x] Basic feature docs are published on preliminary.istio.io describing what the feature does, how to use it, and any caveats.
+- [x] Release notes entries added as appropriate
+- [x] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [x] Automated integration tests cover core use cases with the feature enabled [https://github.com/istio/istio/blob/master/tests/integration/pilot/revisions/revision_tag_test.go](https://github.com/istio/istio/blob/master/tests/integration/pilot/revisions/revision_tag_test.go).
+- [x] When disabled, the feature does not affect system stability or performance.
+
+**API**
+
+- [ ] Initial API review.
```

Comment:
We should do a quick API review of this in TOC, it should be short since the API is small right? Maybe a demo to TOC?